dependencies.yaml

```diff
@@ -106,6 +106,7 @@ dependencies:
         packages:
           - c-compiler
           - cxx-compiler
+          - aws-sdk-cpp
```

Comment:
*question*: Should we set a version constraint on this package? I suspect yes.

Reply:
Yes, we need https://github.com/aws/aws-sdk-cpp/pull/2710